### DISCUSSION TITLE
What country is home to the world's widest avenue?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#655] [Galaxy Trivia] [answered]
**Bonus:** How wide is it?

**Comment by DevElkami** [comment]
The widest avenue in the world is the Avenida 9 de Julio, Buenos Aires, Argentina.

Its span of around 110 m, 16 lanes of traffic.

**Comment by peterbaccaro** [comment, marked as answer]
The country that is home to the world's widest avenue is **Argentina**, and the avenue is Avenida 9 de Julio (9th of July Avenue) in Buenos Aires. It is named after Argentina's Independence Day, which is celebrated on July 9th. The avenue is approximately **140 meters (459 feet) wide** and has up to seven lanes of traffic in each direction, with a central pedestrian boulevard in the middle.

**Comment by FleetAdmiralJakob** [comment]
The widest avenue in the world is Avenida 9 de Julio (9th of July Avenue), located in Buenos Aires, the capital city of Argentina. The avenue is named in honor of Argentina's Independence Day on July 9, 1816, and it measures approximately 140 meters (459 feet) in width, with multiple lanes in each direction and a central pedestrian walkway.
